Tasks/BuildctlV0/Strings/resources.resjson/ko-KR/resources.resjson

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Strings are referring to kubectl tool. This is not in alignment with en-us version of resources.json